lib/redux/onboarding/onboarding_progress_reducer.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:solarisdemo/redux/onboarding/onboarding_progress_action.dart';
+
+import 'onboarding_progress_state.dart';
+
+OnboardingProgressState onboardingProgressReducer(OnboardingProgressState currentState, dynamic action) {
+  if (action is GetOnboardingProgressCommandAction) {
```

Comment:
nu cred ca inteleg de ce tre' sa schimbi state-ul pt actionul asta

Reply:
Am pus ca sa pot afisa un loading pe stepper screen. 
Daca nu schimb state-ul, cand userul ajunge **din nou** pe stepper screen o sa vada informatiile care sunt deja in state (in timpul asta facandu-se un request in spate care o sa schimbe state-ul)